src/plugins/gui/scenes/FiatPluginEnterAmountScene.tsx

```diff
@@ -275,6 +306,10 @@ const getStyles = cacheStyles((theme: Theme) => {
       paddingTop: theme.rem(0.5),
       width: '100%'
     },
+    maxButton: {
+      alignItems: 'flex-end',
+      marginTop: theme.rem(-0.75)
```

Comment:
Defeats the spirit and purpose of `ButtonsViewUi4` but I get this is a temporary one-off thing until we do a proper design. Could have just used a loose button on its own with styling:
```
      flexDirection: 'row',
      justifyContent: 'flex-end',
      marginTop: theme.rem(-0.75)
```

Reply:
I'm only modifying the top margin. Everything else stays the same as `ButtonsUiView`

Reply:
The margins of `ButtonsViewUi4` is a large part of what the component is intended to be responsible for defining and managing, and what I spent a lot of time making happen app-wide. Adding new callers who arbitrarily modify and remove them is moving a step backwards in updating our app to UI4 and the advantages of reuse and consistency. 

Would prefer to have a button in a plain view instead as I described so we can more easily keep track of where we need to define new `ButtonsViewUi4` layouts in case the final scene design takes a while to happen.

We need a complete design for the scene and a corresponding specific, codified layout such as `'right-justified'` for this anyway so approving for now so we can merge asap to revisit later.